Configuration/Generator/python/TenE_E_0_200_pythia8_cfi.py

```diff
@@ -0,0 +1,22 @@
+import FWCore.ParameterSet.Config as cms
+
+# Modified from Configuration/Generator/python/TenMuE_0_200_pythia8_cfi.py
+generator = cms.EDFilter("Pythia8EGun",
```

Comment:
I'm confused by your comment, that name is not the name of the module
(and actually, I checked and I saw all fragments are using "generator" as name and I find it as expected, actually)
but it is the name of the CMSSW template which defines the type of EDFilter we want to run ...